Source/WebKit/UIProcess/API/Cocoa/WKWebExtension.mm

```diff
@@ -176,137 +176,137 @@ - (instancetype)_initWithResources:(NSDictionary<NSString *, id> *)resources
 
 - (NSDictionary<NSString *, id> *)manifest
 {
-    return _webExtension->manifest();
+    return [self _protectedWebExtension]->manifest();
```

Comment:
All of these calls should be `self. _protectedWebExtension`.

Reply:
Fixed.